addons/mail/static/src/components/activity/activity.xml

```diff
@@ -2,52 +2,55 @@
-                    <div class="o_Activity_user">
+                <div class="o_Activity_sidebar mr-3">
+                    <div class="o_Activity_user position-relative">
                         <t t-if="activityView.activity.assignee">
-                            <img class="o_Activity_userAvatar" t-attf-src="/web/image/res.users/{{ activityView.activity.assignee.id }}/avatar_128" t-att-alt="activityView.activity.assignee.nameOrDisplayName"/>
+                            <img class="o_Activity_userAvatar rounded-circle w-100" t-attf-src="/web/image/res.users/{{ activityView.activity.assignee.id }}/avatar_128" t-att-alt="activityView.activity.assignee.nameOrDisplayName"/>
                         </t>
-                        <div class="o_Activity_iconContainer"
+                        <div class="o_Activity_iconContainer d-flex align-items-center justify-content-center rounded-circle w-50 h-50 text-white"
                             t-att-class="{
                                 'bg-success': activityView.activity.state === 'planned',
                                 'bg-warning': activityView.activity.state === 'today',
                                 'bg-danger': activityView.activity.state === 'overdue',
                             }"
                         >
-                            <i class="o_Activity_icon fa" t-attf-class="{{ activityView.activity.icon }}"/>
+                            <i class="o_Activity_icon fa small" t-attf-class="{{ activityView.activity.icon }}"/>
                         </div>
                     </div>
                 </div>
                 <div class="o_Activity_core">
-                    <div class="o_Activity_info">
-                        <div class="o_Activity_dueDateText"
+                    <div class="o_Activity_info d-flex align-items-baseline">
+                        <div class="o_Activity_dueDateText me-2"
                             t-att-class="{
-                                'o-default': activityView.activity.state === 'default',
-                                'o-overdue': activityView.activity.state === 'overdue',
-                                'o-planned': activityView.activity.state === 'planned',
-                                'o-today': activityView.activity.state === 'today',
+                                'text-danger': activityView.activity.state === 'overdue',
+                                'text-success': activityView.activity.state === 'planned',
+                                'text-warning': activityView.activity.state === 'today',
                             }"
                         >
-                            <t t-esc="activityView.delayLabel"/>
+                            <b t-esc="activityView.delayLabel"/>
                         </div>
                         <t t-if="activityView.activity.summary">
-                            <div class="o_Activity_summary">
+                            <b class="o_Activity_summary me-2 h6">
```

Comment:
As I posted on discord, these `h6` are making the bottom line wrongly aligned on the avatar.